editor.planx.uk/package.json

```diff
@@ -25,7 +25,7 @@
     "@tiptap/extension-history": "^2.0.3",
     "@tiptap/extension-image": "^2.0.3",
     "@tiptap/extension-italic": "^2.0.3",
-    "@tiptap/extension-link": "^2.0.3",
+    "@tiptap/extension-link": "^2.1.13",
```

Comment:
This was the issue that was giving me so much grief (causing E2E tests to fail) - I managed to recreate on the Pizza whilst testing.

I'll pick up a PR to revert https://github.com/theopensystemslab/planx-new/pull/2589 shortly.